src/ipc/get-printer-info.ts

```diff
@@ -20,14 +31,26 @@ export async function getPrinterInfo(
     printerSchemes(printers),
   )
 
+  // CUPS makes an IPP server available for each USB printer, starting with port
```

Comment:
FYI this is the function I added retries to in https://github.com/votingworks/kiosk-browser/pull/94/files so you probably want to pull main to make sure you have those changes